docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Rephrase to the following:</p><p> </p><p>In this notebook, we are using two datasets with different resolutions (480km and 120km) from the MPAS Ocean Model. We will be remapping the <code>bottomDepth</code> variable, which measures the ocean depth.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>